client/src/app/core/core-services/stable.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+
+import { BehaviorSubject, Observable } from 'rxjs';
+
+import { Deferred } from '../promises/deferred';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class StableService {
```

Comment:
Integrate this into the OpenSlidesStatusService. Use this service only, so remove the `isStable` in the OpenSlides Service